addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -212,19 +217,44 @@ class AwsCfnService extends Service {
       { action: 'check-aws-permissions', conditions: [allowIfActive, allowIfAdmin] },
       { accountId },
     );
+    // see https://github.com/rvedotrc/aws-cloudformation-stack-states for all states
+    const PENDING_STATES = [
```

Comment:
These states should be defined at file or class level since they're constants used by multiple methods.

Reply:
👍🏻 makes sense